upt_macports/templates/base.Portfile

```diff
@@ -5,24 +5,23 @@ PortSystem          1.0
 {% endblock %}
 
 {% block nameversion %}
-name                {{ pkg._pkgname() }}
-version             {{ pkg.upt_pkg.version }}
 {% endblock %}
+
 maintainers         nomaintainer
 description         {{ pkg.upt_pkg.summary }}
 long_description    ${description}
 
 platforms           darwin
+supported_archs     noarch
 
-{% block versions %}
+{% block dist_info %}
 {% endblock %}
 
-{% block build_deps %}
-{{ depends_build-append pkg.build_depends }}
-{% endblock %}
-{% block run_deps %}
-{{ depends_run-append pkg.run_depends }}
-{% endblock %}
-{% block test_deps %}
-{{ depends('TEST', pkg.test_depends) }}
+license             {% for i in pkg.upt_pkg.licenses %}{{i.name}}{% endfor %}
```

Comment:
I tested `buildbot-pkg`, for example, and nothing is printed at all. If the licence is not known (I assume that is the case when `pkg.upt_pkg.licences` is empty), just output the string `unknown`.